src/Main.java

```diff
@@ -1,6 +1,82 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
+//task1
+        System.out.println("task1");
+        int[] arrInt = new int[]{1, 2, 3};
```

Comment:
этот способ создания аналогичен
```suggestion
        int[] arrInt = {1, 2, 3};
```
предполагалось создать массив длины 3 и заполнить его отдельно, как в подсказке показано